samples/Patient and Population Services G10/infra/core/apiManagement/policies/cacheBlockToken.xml

```diff
@@ -0,0 +1,34 @@
+<!--
+    IMPORTANT:
+    - Policy elements can appear only within the <inbound>, <outbound>, <backend> section elements.
+    - To apply a policy to the incoming request (before it is forwarded to the backend service), place a corresponding policy element within the <inbound> section element.
+    - To apply a policy to the outgoing response (before it is sent back to the caller), place a corresponding policy element within the <outbound> section element.
+    - To add a policy, place the cursor at the desired insertion point and select a policy from the sidebar.
+    - To remove a policy, delete the corresponding policy statement from the policy document.
+    - Position the <base> element within a section element to inherit all policies from the corresponding section element in the enclosing scope.
+    - Remove the <base> element to prevent inheriting policies from the corresponding section element in the enclosing scope.
+    - Policies are applied in the order of their appearance, from the top down.
+    - Comments within policy elements are not supported and may disappear. Place your comments between policy elements or at a higher level scope.
+-->
+<policies>
+    <inbound>
+        <base />
+        <set-variable name="uti" value="@{
+            var request = context.Request.Body.As<JObject>();
+            return request["uti"].ToString();
+        }" />
+        <cache-store-value key="access-@{uti}" value="true" duration="5000" caching-type="prefer-external" />
```

Comment:
Is 'duration' in milliseconds? 5000 seems a little short. Also, in the future should we calculate duration from nbf not to exceed exp?

Reply:
Changed to 6000 as access tokens last 90 mins max.

https://learn.microsoft.com/en-us/azure/active-directory/develop/active-directory-configurable-token-lifetimes#access-tokens